lib/common/mongoose.decorators.ts

```diff
@@ -2,6 +2,7 @@ import { Inject } from '@nestjs/common';
 import { getConnectionToken, getModelToken } from './mongoose.utils';
 
 export const InjectModel = (model: string) => Inject(getModelToken(model));
```

Comment:
Similarly to `@nestjs/typeorm`, can we just accept a second argument here (instead of creating a new decorator)?

Reply:
@kamilmysliwiec I was worried about backwards compatibility concerns, but it's perfectly possible.